apps/files/src/components/Collaborators/SharedFilesList.vue

```diff
@@ -1,12 +1,65 @@
+<template>
+  <file-list :fileData="fileData" id="files-list" :loadingFolder="loadingFolder" :actions="actions" :compactMode="_sidebarOpen"
+    :isActionEnabled="isActionEnabled">
+    <template #headerColumns>
+      <oc-table-cell type="head" class="uk-text-truncate" v-translate>Name</oc-table-cell>
+      <oc-table-cell shrink type="head" class="uk-text-nowrap" v-text="sharedCellTitle" />
+      <oc-table-cell
+        v-if="$route.name === 'files-shared-with-me'"
+        shrink
+        type="head"
+        class="uk-text-nowrap"
+        v-translate
+      >
+        Status
+      </oc-table-cell>
+      <oc-table-cell shrink type="head" class="uk-text-nowrap" v-translate>Share time</oc-table-cell>
+    </template>
+    <template v-slot:rowColumns="{ item }">
+      <oc-table-cell class="uk-text-truncate">
+        <oc-file @click.native.stop="item.type === 'folder' ? navigateTo(item.path.substr(1)) : openFileActionBar(item)"
+          :name="item.basename" :extension="item.extension" class="file-row-name" :icon="fileTypeIcon(item)"
+          :filename="item.name" :key="item.path" />
+        <oc-spinner
+          v-if="actionInProgress(item)"
+          size="small"
+          :uk-tooltip="disabledActionTooltip(item)"
+          class="uk-margin-small-left"
+        />
+      </oc-table-cell>
+      <oc-table-cell class="uk-text-meta uk-text-nowrap">
+        <div v-if="$route.name === 'files-shared-with-others'" key="shared-with-cell">
+          {{ item.sharedWith }} <translate v-if="item.shareType === 1">(group)</translate>
```

Comment:
I know this was already also before but maybe would be better for translations to put the parentheses outside of the translate tag so translators only translate "group"?